src/tests/multihost/basic/conftest.py

```diff
@@ -25,23 +27,39 @@ def pytest_namespace():
 
 
 @pytest.fixture(scope="class")
-def multihost(session_multihost, request):
+def multihost(session_multihost):
+    """ multihost fixture """
     return session_multihost
 
 
 @pytest.fixture(scope="session")
-def run_authselect(session_multihost, request):
+def package_install(session_multihost):
+    """ Install required packages """
+    distro = session_multihost.master[0].distro
+    pkg_list = 'authselect nss-tools 389-ds-base krb5-server'\
+               'openldap-clients krb5-workstation '\
+               '389-ds-base-legacy-tools sssd sssd-kcm'
```

Comment:
Later on, the sssd configuration sets up the ifp service. But with recent versions of sssd, just installing the sssd meta-package doesn't install sssd-dbus which provides the ifp service. So either you need to add sssd-dbus as another package to install or not configure the sssd-ifp service.